broker/broker-security-api/src/main/java/org/codice/ddf/broker/security/api/SecurityServerPlugin.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.broker.security.api;
+
+import org.apache.activemq.artemis.api.core.Message;
+import org.apache.activemq.artemis.core.server.ServerSession;
+import org.apache.activemq.artemis.core.transaction.Transaction;
+
+public interface SecurityServerPlugin {
```

Comment:
This interface bothers me a bit, I understand why we need it but the name and method aren't very descriptive to what its intended use is.

Reply:
I left the method "beforeSend" because that's what it is in the Artemis plugin that i'm being wrapped by. I'm certainly open to renaming either the class, method or both though. Do you have any ideas?

Reply:
Fixed